createdisk.sh

```diff
@@ -185,7 +185,8 @@ if [ $BUNDLE_TYPE != "microshift" ]; then
     create_qemu_image "$libvirtDestDir" "${VM_PREFIX}-base" "${VM_NAME}"
     mv "${libvirtDestDir}/${VM_NAME}" "${libvirtDestDir}/${SNC_PRODUCT_NAME}.qcow2"
 else
-    sparsify_lvm "${libvirtDestDir}"
+    create_qemu_image "$libvirtDestDir" "${VM_NAME}.qcow2" "microshift"
+    mv "${libvirtDestDir}/microshift" "${libvirtDestDir}/${SNC_PRODUCT_NAME}.qcow2"
```

Comment:
if `VM_NAME` is different from `SNC_PRODUCT_NAME`, it's probably possible to directly pass `SNC_PRODUCT_NAME.qcow2` to `create_qemu_image`

Reply:
@cfergeau in case of microshift it is same https://github.com/crc-org/snc/pull/750/files#diff-030b04caaa878905ffc5b7caa7e11fbae90df58669834ebe42fbbe907bfb6db7R41-R46 so I used `${VM_NAME}`